test/ccd2rgb.jl

```diff
@@ -0,0 +1,48 @@
+@testset "ccd2rgb" begin
+    if !isfile(joinpath("data","casa_0.5-1.5keV.fits"))
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_0.5-1.5keV.fits", joinpath("data","casa_0.5-1.5keV.fits"))
+    end
+    if !isfile(joinpath("data","casa_1.5-3.0keV.fits"))
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_1.5-3.0keV.fits", joinpath("data","casa_1.5-3.0keV.fits"))
+    end
+    if !isfile(joinpath("data","casa_4.0-6.0keV.fits"))
+        mkpath("data")
+        download("http://chandra.harvard.edu/photo/2009/casa/fits/casa_4.0-6.0keV.fits", joinpath("data","casa_4.0-6.0keV.fits"))
+    end
+    if !isfile(joinpath("data","ccd2rgb.jld"))
+        mkpath("data")
+        download("https://bintray.com/aquatiko/AstroImages.jl/download_file?file_path=ccd2rgb.jld",joinpath("data","ccd2rgb.jld"))
+    end
+        
+    r = FITS(joinpath("data","casa_0.5-1.5keV.fits"))[1]
+    b = FITS(joinpath("data","casa_1.5-3.0keV.fits"))[1]
+    g = FITS(joinpath("data","casa_4.0-6.0keV.fits"))[1]
+    linear_res = RGB.(ccd2rgb(r, b, g, shape_out = (1000,1000)))
+    asinh_res = RGB.(ccd2rgb(r, b, g, shape_out = (1000,1000), stretch = asinh))
+    
+    linear_ans = load(joinpath("data","ccd2rgb.jld"), "linear")
+    asinh_ans = load(joinpath("data","ccd2rgb.jld"), "asinh")
+    
+    function check_diff(arr1, arr2)
+        diff = 1e-4
+        count = 0
+        for i in 1 : size(arr1)[1]
+            for j in 1 : size(arr1)[2]
+                if abs(arr1[i,j] - arr2[i,j]) > diff
```

Comment:
See this: here if won't exceed a count if one or both of the number is a NaN. Thus, the condition of `isnan` is satisfied in a way.

Reply:
This is completely wrong: if one of the two is `NaN`, the test should fail, if both are `NaN` the test should pass:
```julia
julia> NaN - 0.5 > 1e-4
false

julia> NaN - NaN > 1e-4
false

julia> isapprox(NaN, 0.5, nans = true)
false

julia> isapprox(NaN, NaN, nans = true)
true
```